index.html

```diff
@@ -0,0 +1,142 @@
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary sticky-top">
+        <a class="navbar-brand" href="./">
+            <img src="img/favicon.png" class="navbrand-img" alt="lint icon">
+            Java Style Linter
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="nav ml-auto nav-tabs">
+                <li class="nav-item"><a class="nav-link " href="splash.html" id="splash-link">About</a></li>
+                <li class="nav-item"><a class="nav-link active" href="#" id="main-link">Style Checker</a></li>
+                <li class="nav-item"><a class="nav-link" href="https://github.com/info340c-sp20/project-1-oibra">Codebase</a></li>
+            </ul>
+        </div>
+    </nav>
+    <main id="main-page">
+        <div class="container">
+            <section class="mt-3">
+                <div class="container">
+                    <h1>Java Style Linter</h1>
+                    <p>This web app functions as an educational Java linter that checks the style of users’ input Java code and inform users of what stylistic issues the code has with suggestions on how to fix them. This app is built around the style guidelines for UW CSE 14X (Intro CS1 &amp; 2 at UW) with those students as the primary audience, but should be a helpful resource for any novice Java learners. Be aware though, that if you are not a student in those classes, make sure that your code fits the quality guidelines of your organization and that they're consistent with the guidelines used by this tool if you want to use it.</p>
+                    <div class="alert alert-warning">
+                        This tool is designed to work on compiling code. If the input code wouldn't compile, I make no guarantees about the accuracy of the results.
+                    </div>
+                </div>
+            </section>
+            <div class="container">
+                <h2 class="collapse-toggler" type="button" role="button" data-toggle="collapse" data-target="#code-form">
+                    <img class="collapse-button open" src="img/expand_more.svg" alt="collapse button"/>
+                    Code Input
+                </h2>
+                <section class="collapse show" id="code-form">
+                    <form action="./js/index.js" method="POST" spellcheck="false">
```

Comment:
I don't think it's possible to POST to a JS file, so you don't need this action and method